common/shlibs

```diff
@@ -16,6 +16,13 @@
 # PLEASE NOTE: when multiple packages provide the same SONAME, the first
 # one (order top->bottom) is preferred over the next ones.
 #
+libtss2-mu.so.0 tpm2-tss-2.3.1_1
```

Comment:
Those so maybe important to either _you_ or fwupd.
But it's not as important as other package, especially libc.

Better to move it down the list, last of the list should be better.

Reply:
I'll move it down in next revision